apps/api/src/domain/models/digital-text/digital-text-page.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { NonEmptyString } from '@coscrad/data-types';
+import { DTO } from '../../../types/DTO';
+import BaseDomainModel from '../BaseDomainModel';
+import { PageIdentifier } from './page-identifier';
+
+export default class DigitalTextPage extends BaseDomainModel {
+    @NonEmptyString({
+        label: 'identifier',
+        description: 'text identifier for the page',
+    })
+    readonly identifier: PageIdentifier;
+
+    constructor(dto: DTO<DigitalTextPage>) {
+        super();
+
+        if (!dto) return;
+
+        const { identifier } = dto;
+
+        this.identifier = identifier;
```

Comment:
Comment that this is really just a string. Were it a reference value, you'd need to clone.

Reply:
done